products/satellite.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Red Hat Satellite Server
+category: server-app
+changelogTemplate: "https://access.redhat.com/documentation/en-us/red_hat_satellite/__RELEASE_CYCLE__/html/release_notes/index"
+releases:
+-   releaseCycle: "6.12"
+    eol: 2024-05-31
+    support: 2023-05-31
+    releaseDate: 2022-11-16
+    latest: "6.12.0"
+    latestReleaseDate: 2022-11-16
+-   releaseCycle: "6.11"
+    eol: 2024-01-31
+    support: 2022-11-30
+    releaseDate: 2022-07-05
+    latest: "6.11.4.1"
+    latestReleaseDate: 2022-11-17
+-   releaseCycle: "6.10"
+    eol: 2023-05-31
+    support: 2022-06-30
+    releaseDate: 2021-11-16
+    latest: "6.10.7.1"
+    latestReleaseDate: 2022-11-17
+-   releaseCycle: "6.9"
+    eol: 2022-11-30
+    support: 2021-11-30
+    releaseDate: 2021-11-16
+    latest: "6.9.10"
+    latestReleaseDate: 2022-11-17
+iconSlug: redhat
+permalink: /satellite
+alternate_urls:
+-   /rhsat
```

Comment:
Maybe also `/redhat-satellite` ?

Reply:
I changed to permalink to redhat-satellite, is this what you intended? Or do you mean the alternative_urls?

Reply:
I initially meant the alternate_urls, but that's fine too.